src/editor/views/gridstackPanel.ts

```diff
@@ -36,6 +24,10 @@ export class GridStackPanel extends Widget {
     this._cells = new Map<string, GridItem>();
   }
 
+  get cells(): Map<string, GridItem> {
```

Comment:
Adding this getter for now, but should be removed in the next refactoring.

Related to https://github.com/hbcarlos/voila-editor/issues/48